SICAR/sicar.py

```diff
@@ -62,6 +64,38 @@ def __init__(
         self._create_session(headers=headers)
         self._initialize_cookies()
 
+    @staticmethod
+    def _parse_update_date_html(html_response) -> Dict:
```

Comment:
```suggestion
    def _parse_release_date(response) -> Dict:
```

Could you please try to annotate `response`?

Reply:
also did not see this comment, I will update code and push again